src/connected/index.ts

```diff
@@ -7,4 +7,5 @@ export * from './export-only/exportableState';
 export * from './dataUtils';
 export * from './loaderUtils';
 export * from './model/constants';
+export * from './model/state';
```

Comment:
Oops, a previous comment I left was pretty misleading. This file should _not_ be exported; I don't want to expose `ChartState` or another of the other internal types as part of the public API (see [`ChartProviderState`](https://github.com/palantir/react-layered-chart/blob/master/API.md#chartproviderstate)). This was my mistake leaving a very leading comment on the example code you changed; could you change it to use `ChartProviderState` instead?